tests/numpy/test_misc.py

```diff
@@ -121,12 +121,102 @@ def test_all(nps_app_inst):
     assert nps.all(false_float).dtype is np.bool
 
 
+def test_alltrue(nps_app_inst):
+    import nums.numpy as nps
+    assert nps_app_inst is not None
+
+    true_int = nps.array([[1, 2, 3], [1, 2, 3]]).reshape(block_shape=(2, 2))
+    false_int = nps.array([[1, 2, 0], [1, 2, 3]]).reshape(block_shape=(2, 2))
+    true_bool = nps.array([True, True])
+    false_bool = nps.array([True, False])
+    true_float = nps.array([[1.0, 2.0, 3.0], [1.0, 2.0, 3.0]]).reshape(block_shape=(2, 2))
+    false_float = nps.array([[1.0, 2.0, 0.0], [1.0, 2.0, 3.0]]).reshape(block_shape=(2, 2))
+
+    assert nps.alltrue(true_int)
+    assert not nps.alltrue(false_int)
+    assert nps.alltrue(true_bool)
+    assert not nps.alltrue(false_bool)
+    assert nps.alltrue(true_float).get() == \
+           np.alltrue(np.array([[1.0, 2.0, 3.0], [1.0, 2.0, 3.0]]))
+    assert nps.alltrue(false_float).get() == \
+           np.alltrue(np.array([[1.0, 2.0, 0.0], [1.0, 2.0, 3.0]]))
+
+    assert nps.alltrue(true_int).dtype is np.bool
+    assert nps.alltrue(false_int).dtype is np.bool
+    assert nps.alltrue(true_bool).dtype is np.bool
+    assert nps.alltrue(false_bool).dtype is np.bool
+    assert nps.alltrue(true_float).dtype is np.bool
+    assert nps.alltrue(false_float).dtype is np.bool
+
+
+def test_any(nps_app_inst):
```

Comment:
This looks very similar to `all` and `alltrue`. Can we consolidate the code for these tests?

Reply:
Yep!